dev/benchmark/sysbench-in-k8s.md

```diff
@@ -0,0 +1,452 @@
+---
+title: TiDB in Kubernetes Sysbench 性能测试
+category: benchmark
+---
+
+# TiDB in Kubernetes Sysbench 性能测试
+
+随着 [TiDB Operator GA 发布](https://pingcap.com/blog-cn/tidb-operator-1.0-ga/)，越来越多用户开始使用 TiDB Operator 在 Kubernetes 中部署管理 TiDB 集群。在本次测试中，我们选择 GKE 平台做了一次深入、全方位的测试，方便大家了解 TiDB 在 Kubernetes 中性能影响因素。
+
+## 目的
+
+- 测试典型公有云平台上 TiDB 性能数据
+- 测试公有云平台磁盘、网络、CPU 以及不同 Pod 网络下对 TiDB 性能的影响
+
+## 环境
+
+### 版本与配置
+
+本次测试统一使用 TiDB v3.0.1 版本进行测试。
+
+TiDB Operator 使用 v1.0.0 版本。
+
+PD、TiDB 和 TiKV 实例数均为 3 个。各组件分别作了如下配置，未配置部分使用默认值。
+
+PD:
+
+```toml
+[log]
+level = "info"
+[replication]
+location-labels = ["region", "zone", "rack", "host"]
+```
+
+TiDB:
+
+```toml
+[log]
+level = "error"
+[prepared-plan-cache]
+enabled = true
```

Comment:
max-procs is not set according to limits.cpu?

Reply:
one tidb instance per node, so no need to set max-procs